Controls/LoadablePanel/src/DotVVM.Contrib/LoadablePanel.cs

```diff
@@ -72,15 +72,28 @@ public bool HideUntilLoaded
             DotvvmProperty.Register<bool, LoadablePanel>(t => t.HideUntilLoaded);
 
         /// <summary>
-        /// Gets or sets a collection of values of all loadable panels which are currently loading.
+        /// Gets or sets a collection of element ids of all loadable panels which are currently in the loading state.
         /// </summary>
-        public IEnumerable LoadingItems
+        public IEnumerable LoadingElementIds
```

Comment:
@Mylan719 I'm not sure whether we use "Ids" anywhere. Please rename LoadingElementIds to LoadingKeySource.